libs/math/src/math/linalg/blas/gemv_t_vector.cpp

```diff
@@ -166,12 +166,26 @@ template class Blas<double, Signature(_y <= _alpha, _A, _x, _n, _beta, _y, _m),
 template class Blas<float, Signature(_y <= _alpha, _A, _x, _n, _beta, _y, _m),
                     Computes(_y <= _alpha * T(_A) * _x + _beta * _y),
                     platform::Parallelisation::VECTORISE>;
+template class Blas<
```

Comment:
and here also?

Reply:
fp128 does not -yet- support vectorisation, so it's intentionally omitted.